etcdctl/ctlv3/command/del_command.go

```diff
@@ -39,6 +42,7 @@ func NewDelCommand() *cobra.Command {
 	cmd.Flags().BoolVar(&delPrefix, "prefix", false, "delete keys with matching prefix")
 	cmd.Flags().BoolVar(&delPrevKV, "prev-kv", false, "return deleted key-value pairs")
 	cmd.Flags().BoolVar(&delFromKey, "from-key", false, "delete keys that are greater than or equal to the given key using byte compare")
+	cmd.Flags().BoolVar(&delRange, "range", false, "delete range of keys")
```

Comment:
If I understand @ptabor 's comment correctly, we don't need add any flag for 3.5. Instead, we just need to add a warning message.

Reply:
https://github.com/etcd-io/etcd/pull/13729#discussion_r814548212
In this comment. ptabor support add a flag `--range` to ingore this message.